salt/cloud/clouds/ec2.py

```diff
@@ -138,14 +138,16 @@
 
 EC2_LOCATIONS = {
     'ap-northeast-1': 'ec2_ap_northeast',
+    'ap-northeast-2': 'ec2_ap_northeast_2',
     'ap-southeast-1': 'ec2_ap_southeast',
     'ap-southeast-2': 'ec2_ap_southeast_2',
     'eu-west-1': 'ec2_eu_west',
     'eu-central-1': 'ec2_eu_central',
-    'sa-east-1': 'ec2_sa_east',
+    'us-gov-west-1': 'ec2_us_gov_west_1’,
     'us-east-1': 'ec2_us_east',
     'us-west-1': 'ec2_us_west',
     'us-west-2': 'ec2_us_west_oregon',
+    'sa-east-1': 'ec2_sa_east',
```

Comment:
Any particular reason this line was moved to the end? I think these were listed in alphabetical order so we can keep track of things as the list grows.